### DISCUSSION TITLE
Octernship Application Issue!

### DISCUSSION THREAD

**Discussion by kirtanchandak** [community/community#51014] [GitHub Education]
Hey, I am not to apply for more than one octernship. Currently I have applied only for one. We can apply 4 at a time I as written on the page. 
But when I applying I am getting an error.  
![Screenshot 2023-03-25 205516](https://user-images.githubusercontent.com/92196705/227726667-ebfb4f5a-2dd7-4c93-9226-c5740f3d74a5.png)

**Comment by vanshika-singh518** [comment]
It is because the number of applicants have reached the max limit for that octernship. Try on a different one that you might be interested in and it'll work. For some the limit is around 500.

**Comment by ghoshvidip26** [comment]
What is the last date for the submission of Octernship applications?
